webcomponents-hi-sd-ce.js

```diff
@@ -1,6 +1,6 @@
ment,'currentScript',{get(){return wt||('complete'===document.readyState?null:document.scripts[document.scripts.length-1])},configurable:!0});const Dt=/(^\/)|(^#)|(^[\w-\d]*:)/,Lt=/(url\()([^)]*)(\))/g,kt=/(@import[\s]+(?!url\())([^;]*)(;)/g,Ht=/(<link[^>]*)(rel=['|"]?stylesheet['|"]?[^>]*>)/g,Ft={fixUrls(ro,no){ro.href&&ro.setAttribute('href',Ft.replaceAttrUrl(ro.getAttribute('href'),no)),ro.src&&ro.setAttribute('src',Ft.replaceAttrUrl(ro.getAttribute('src'),no)),'style'===ro.localName&&Ft.resolveUrlsInStyle(ro,no)},fixUrlAttributes(ro,no){const so=['action','src','href','url','style'];for(let lo,ao=0;ao<so.length&&(lo=so[ao]);ao++){const io=ro.attributes[lo],po=io&&io.value;po&&0>po.search(/({{|\[\[)/)&&(io.value='style'===lo?Ft.resolveUrlsInCssText(po,no):Ft.replaceAttrUrl(po,no))}},fixUrlsInTemplates(ro,no){const so=ro.querySelectorAll('template');for(let ao=0;ao<so.length;ao++)Ft.fixUrlsInTemplate(so[ao],no)},fixUrlsInTemplate(ro,no){const so=ro.content||ro,ao=so.querySelectorAll('style, form[action], [src], [href], [url], [style]');for(let lo=0;lo<ao.length;lo++){const io=ao[lo];'style'==io.localName?Ft.resolveUrlsInStyle(io,no):Ft.fixUrlAttributes(io,no)}Ft.fixUrlsInTemplates(so,no)},resolveUrlsInStyle(ro,no){ro.textContent=Ft.resolveUrlsInCssText(ro.textContent,no)},resolveUrlsInCssText(ro,no){let so=Ft.replaceUrls(ro,no,Lt);return so=Ft.replaceUrls(so,no,kt),so},replaceUrls(ro,no,so){return ro.replace(so,(ao,lo,io,po)=>{let co=io.replace(/["']/g,'');return no&&(co=Ft.resolveUrl(co,no)),lo+'\''+co+'\''+po})},replaceAttrUrl(ro,no){return ro&&Dt.test(ro)?ro:Ft.resolveUrl(ro,no)},resolveUrl(ro,no){if(void 0===Ft.__workingURL){Ft.__workingURL=!1;try{const ao=new URL('b','http://a');ao.pathname='c%20d',Ft.__workingURL='http://a/c%20d'===ao.href}catch(ao){}}if(Ft.__workingURL)return new URL(ro,no).href;let so=Ft.__tempDoc;return so||(so=document.implementation.createHTMLDocument('temp'),Ft.__tempDoc=so,so.__base=so.createElement('base'),so.head.appendChild(so.__base),so.__anchor=so.createElement('a')),so.__base.href=no,so.__anchor.href=ro,so.__anchor.href||ro}},Ut={async:!0,load(ro){return new Promise((no,so)=>{if(!ro)so({resource:'error: href must be specified'});else if(ro.match(/^data:/)){const ao=ro.split(','),lo=ao[0];let io=ao[1];io=-1<lo.indexOf(';base64')?atob(io):decodeURIComponent(io),no({resource:io})}else{const ao=new XMLHttpRequest;ao.open('GET',ro,Ut.async),ao.addEventListener('readystatechange',()=>{if(4===ao.readyState){let lo;try{const po=ao.getResponseHeader('Location');po&&(lo='/'===po.substr(0,1)?location.origin+po:po)}catch(po){console.error(po.message)}const io={resource:ao.response||ao.responseText,redirectedUrl:lo};304===ao.status||0===ao.status||200<=ao.status&&300>ao.status?no(io):so(io)}}),ao.send()}})}},Wt=/Trident/.test(navigator.userAgent)||/Edge\/\d./i.test(navigator.userAgent),qt='onunhandledrejection'in window,Bt='link[rel=import]',jt='import-disable',Gt=`link[rel=stylesheet][href][type=${jt}]`,Xt=`${Bt}, ${Gt},
```

Comment:
=>, let, const... - not compatible with Firefox, IE 11 (SyntaxError: let is a reserved identifier / SCRIPT1002: Syntax error ): 
may be compile all to honest es5?